src/pipecat/services/gemini_multimodal_live/vertex.py

```diff
@@ -0,0 +1,187 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+"""Google Vertex AI Gemini Multimodal Live service.
+
+This module provides integration with Google's Gemini Multimodal Live model via
+Vertex AI, supporting both text and audio modalities with voice transcription,
+streaming responses, and tool usage.
+"""
+
+import json
+import os
+from typing import List, Optional, Union
+
+from loguru import logger
+
+from pipecat.adapters.schemas.tools_schema import ToolsSchema
+from pipecat.services.gemini_multimodal_live.gemini import (
+    GeminiMultimodalLiveLLMService,
+    HttpOptions,
+    InputParams,
+)
+
+try:
+    from google.auth import default
+    from google.auth.exceptions import GoogleAuthError
+    from google.auth.transport.requests import Request
+    from google.genai import Client
+    from google.oauth2 import service_account
+
+except ModuleNotFoundError as e:
+    logger.error(f"Exception: {e}")
+    logger.error(
+        "In order to use Google Vertex AI, you need to `pip install pipecat-ai[google]`. Also, set up your Google credentials properly."
+    )
+    raise Exception(f"Missing module: {e}")
+
+
+class GeminiVertexMultimodalLiveLLMService(GeminiMultimodalLiveLLMService):
+    """Google Vertex AI Gemini Multimodal Live service.
+
+    Provides access to Google's Gemini Multimodal Live model via Vertex AI,
+    supporting both text and audio modalities. It handles voice transcription,
+    streaming audio responses, and tool usage.
+    """
+
+    def __init__(
+        self,
+        *,
+        credentials: Optional[str] = None,
+        credentials_path: Optional[str] = None,
+        location: str = "us-east4",
+        project_id: str,
+        model="google/gemini-2.0-flash-live-preview-04-09",
```

Comment:
This was the model that most documentation still seemed to point to as a good default